esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -201,10 +201,10 @@ void CWsESDLConfigEx::init(IPropertyTree *cfg, const char *process, const char *
     if(servicecfg == NULL)
         throw MakeStringException(-1, "config not found for service %s/%s",process, service);
 
-    if(!EsdlBindingImpl::ensureSDSPath(ESDL_DEFS_ROOT_PATH))
+    if(!ensureSDSPath(ESDL_DEFS_ROOT_PATH))
         throw MakeStringException(-1, "Could not ensure '%s' element on Dali", ESDL_DEFS_ROOT_PATH);
```

Comment:
I would make sense to put this exception in ensureSDSPath and make it a void, since the RTM_CREATE_QUERY should never fail in there anyway.